### DISCUSSION TITLE
Rendering issue

### DISCUSSION THREAD

**Discussion by sumit-geek-del** [flyerhq/flutter_chat_ui#461] [Q&A] [answered]
I have used your package to implement chat feature in my flutter app. Everything is working very smoothly I have only issue that when messages are fetched from API it shows a some delay in rendering the messages again . Like when we send message show new message takes some time to render new message .

**Comment by demchenkoalex** [comment]
Hello, how many messages do you have in your chat? Current version has performance problems..

**Reply by sumit-geek-del** [reply]
I got like 100 messages

**Reply by demchenkoalex** [reply]
Yeah, this is the mark when this breaks :D I am working on v2 that can handle hundreds of thousands of messages without issues, but I have it super simple (text only now) so hopefully I will be able to achieve a good balance between UI/UX/performance. No timelines for this one though, work in progress.

**Comment by demchenkoalex** [comment, marked as answer]
should be fixed with v2 https://github.com/flyerhq/flutter_chat_ui/issues/646
